R/generate_toml.R

```diff
@@ -0,0 +1,93 @@
+merge_named_lists <- function(existing, overwriting,
```

Comment:
You may have a good reason for this, but I would strongly advise refactoring this to just use a standard function like `modifyList` or `purrr::list_merge`. I don't like the maintenance burden of adding such a new function ourselves.

Reply:
Oh, it is just my ignorance. `purrr::list_modify()` does exactly what I need, thanks for catching this!